src/state/index.ts

```diff
@@ -1,6 +1,7 @@
 import { configureStore } from '@reduxjs/toolkit'
 import { load, save } from 'redux-localstorage-simple'
 
+import advancedTradingView from '../services/AdvancedTradingView'
```

Comment:
With the EcoBrige, the reducer is at  `src/services/EcoBridge/store/EcoBridge.reducer`, it would be better to have the reducer for this module at a similar location.

Reply:
fixed [501266b](https://github.com/levelkdev/swapr-dapp/pull/1503/commits/501266b3e2ac4a76be7811d2fa67025ee4d1d58b)